lib/fileCache.js

```diff
@@ -58,7 +58,7 @@ var file_cache = {
         if(typeof request_url !== "undefined") {
             path = path + request_url + '/' + filename;
         } else {
-            path = path + '/' + filename;
+            path = path + '/' + key;
```

Comment:
The `+ '/' + filename` should still be there, otherwise it's inconsistent with the setter ([which does add the `---`](https://github.com/rogierslag/prerender-file-cache/pull/1/files#diff-faaa540227decdf2d9c129f9921375a3R97))